addons/web/static/tests/helpers/mock_server.js

```diff
@@ -538,23 +538,69 @@ var MockServer = Class.extend({
      * @param {Array} args
      * @returns {Object}
      */
-    _mockSearchPanelSelectRange: function (model, args) {
-        var fieldName = args[0];
-        var field = this.data[model].fields[fieldName];
+    _mockSearchPanelSelectRange: function (model, args, kwargs) {
+        const fieldName = args[0];
+        const searchDomain = kwargs.search_domain;
+        const disableCounters = kwargs.disable_counters || false;
+        const field = this.data[model].fields[fieldName];
 
-        if (field.type !== 'many2one') {
-            throw new Error('Only fields of type many2one are handled');
+        if (!['many2one', 'selection'].includes(field.type)) {
+            throw new Error('Only fields of type many2one and selection are handled');
         }
 
-        var fields = ['display_name'];
-        var parentField = this.data[field.relation].fields.parent_id;
-        if (parentField) {
-            fields.push('parent_id');
```

Comment:
Is it always that way, why is it not parentField? I don't see _parent_name or something like that here